packages/zeplin-extension-style-kit/elements/layer.js

```diff
@@ -236,24 +243,136 @@ class Layer {
         return declarations;
     }
 
-    /* eslint-disable complexity */
-    collectDeclarations() {
+    shouldAddWidth() {
         const {
-            elementBorder,
-            object: layer
+            object: {
+                parent,
+                layoutGrow,
+                layoutAlignment
+            }
+        } = this;
+        return !parent || !parent.layout || (
+            parent.layout.direction === "row"
```

Comment:
We can an enum for hard-coded values such as `"row"` here and `"stretch"` and `"column"` below.